tools/ci_build/generate_nuspec_for_native_nuget.py

```diff
@@ -0,0 +1,462 @@
+        nuget_artifacts_dir = Path(args.native_build_path) / "nuget-artifacts"
+        print("nuget_artifacts_dir: ", nuget_artifacts_dir,nuget_artifacts_dir.exists())
+        if nuget_artifacts_dir.exists():
+            # Code path for ADO build pipeline, the files under 'nuget-artifacts' are
+            # downloaded from other build jobs
+
+            generate_file_list(nuget_artifacts_dir, files_list, False)
+            is_ado_packaging_build = True
+        else:
+            # Code path for local dev build
+            files_list.append(
+                "<file src=" + '"' + os.path.join(args.native_build_path, "ortextensions.lib") + runtimes + " />"
+            )
+            files_list.append(
+                "<file src=" + '"' + os.path.join(args.native_build_path, "ortextensions.dll") + runtimes + " />"
+            )
+
+    else:
+        files_list.append(
+            "<file src="
+            + '"'
+            + os.path.join(args.native_build_path, "nuget-staging/usr/local/lib", "libortextensions.so")
+            + '" target="runtimes/linux-'+ args.target_architecture 
+            + '/native" />'
+        )
+
+    if not is_cpu_package:
+        # Process props file
+        source_props = os.path.join(
+            args.sources_path, "csharp", "src", "Microsoft.ML.OnnxRuntime", "targets", "netstandard", "props.xml"
+        )
+        target_props = os.path.join(
+            args.sources_path,
+            "csharp",
+            "src",
+            "Microsoft.ML.OnnxRuntime",
+            "targets",
+            "netstandard",
+            args.package_name + ".props",
+        )
+        os.system(copy_command + " " + source_props + " " + target_props)
+        files_list.append("<file src=" + '"' + target_props + '" target="build\\native" />')
+        files_list.append("<file src=" + '"' + target_props + '" target="build\\netstandard1.1" />')
+        files_list.append("<file src=" + '"' + target_props + '" target="build\\netstandard2.0" />')
+
+        # Process targets file
+        source_targets = os.path.join(
+            args.sources_path, "csharp", "src", "Microsoft.ML.OnnxRuntime", "targets", "netstandard", "targets.xml"
+        )
+        target_targets = os.path.join(
+            args.sources_path,
+            "csharp",
+            "src",
+            "Microsoft.ML.OnnxRuntime",
+            "targets",
+            "netstandard",
+            args.package_name + ".targets",
+        )
+        os.system(copy_command + " " + source_targets + " " + target_targets)
+        files_list.append("<file src=" + '"' + target_targets + '" target="build\\native" />')
+        files_list.append("<file src=" + '"' + target_targets + '" target="build\\netstandard1.1" />')
+        files_list.append("<file src=" + '"' + target_targets + '" target="build\\netstandard2.0" />')
+
+        # Process xamarin targets files
+        if args.package_name == "Microsoft.ML.OnnxRuntime.Extensions_xmamarin":

```

Comment:
The Microsoft.ML.OnnxRuntime.Extensions package should include all the xamarin and maui targets. Trying to avoid multiple nuget packages as it greatly complicates usage.

Reply:
I don't have these files for now. So I would either remove this or have a condition to pass by this branch.